src/api/composition/CompositionCollection.js

```diff
@@ -209,7 +211,13 @@ export default class CompositionCollection {
     this.#cleanUpMutables();
     const children = await this.#provider.load(this.domainObject);
     const childObjects = await Promise.all(
-      children.map((c) => this.#publicAPI.objects.get(c, abortSignal))
+      children.map((c) => {
```

Comment:
iteration element could be named `child` to make it more intuitive.

Reply:
Yep, agree, I'll change it, thanks!